docker/package/tezos_voting_wizard.py

```diff
@@ -142,23 +170,38 @@ def check_baking_service(self):
         net = self.config["network"]
         try:
             proc_call(f"systemctl is-active --quiet tezos-baking-{net}.service")
+            return True
         except:
             print(f"Looks like the tezos-baking-{net} service isn't running.")
-            print("Please start the service or set up baking.")
-            if self.config["network_mode"] == "mainnet":
-                print("You can do this by running:")
-                print("tezos-setup-wizard")
-            sys.exit(1)
+            return False
 
     def check_data_correctness(self):
         print("Baker data detected is as follows:")
         print(f"Data directory: {self.config['client_data_dir']}")
         print(f"Node RPC address: {self.config['node_rpc_addr']}")
         print(f"Baker alias: {self.config['baker_alias']}")
-        if not yes_or_no("Does this look correct? (Y/n)", "yes"):
-            print("Try setting up baking again by running:")
-            print("tezos-setup-wizard")
-            sys.exit(1)
+        return yes_or_no("Does this look correct? (Y/n) ", "yes")
+
+    def collect_baking_info(self):
+        if self.check_baking_service():
+            self.fill_baking_config()
+            collected = self.check_data_correctness()
+        else:
+            # Here, we collect some semi-sensible defaults. Since the baking service isn't
+            # running here, these defaults have a low chance of being correct.
+            print("Default values will be used. Please check them carefully.\n")
+
+            self.config["client_data_dir"] = "/var/lib/tezos/.tezos-client"
+            self.config["node_rpc_addr"] = "http://localhost:8732"
```

Comment:
:thinking: if there is a config file in the data dir (which should be the case) can't this be found in the `endpoint` there (if specified)?